services/dynamodb/src/main/java/software/amazon/awssdk/services/dynamodb/model/ItemBuilder.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.dynamodb.model;
+
+import static java.util.stream.Collectors.toList;
+
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import software.amazon.awssdk.services.dynamodb.utils.AttributeValueHelper;
+import software.amazon.awssdk.utils.builder.SdkBuilder;
+
+public final class ItemBuilder implements SdkBuilder<ItemBuilder, Map<String, AttributeValue>> {
+
+    private Map<String, AttributeValue> item = new HashMap<>();
+
+    private ItemBuilder() {}
+
+    public static ItemBuilder builder() {
+        return new ItemBuilder();
+    }
```

Comment:
Should this be `ItemBuilder.create()`?

Reply:
No this, is a builder.

Reply:
It's a builder for an item. 

It SHOULD be `Item.builder()`, but there's no such thing as an "Item". 

I'd expect that `ItemBuilder.builder()` would return an `ItemBuilder.Builder` and its `build()` method would return an `ItemBuilder`. in this case we're creating an item builder, not an item builder builder, hence `ItemBuilder.create()` being correct to me.

Reply:
Fair - in terms of consistency would it make more sense to have an `Item` object (which would probably just extend `HashMap<String, AttributeValue>`) with an `Item.builder()` method that returns (as you say) an `Item.Builder` - this would be consistent with the rest of our models.